### DISCUSSION TITLE
Can't Signup for purchase pro licence.

### DISCUSSION THREAD

**Discussion by AkkiaS7** [reloadware/reloadium#202] [General]
haven.reloadware.com
Have an untreated cert {Kubernetes Ingress Controller Fake Certificate}
And wrong CORS setting {Access to fetch at 'https://haven.reloadware.com/graphql/' from origin 'https://reloadium.io' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.}

**Comment by dkrystki** [comment]
Hi @AkkiaS7 
Reloadium Pro is free now
